android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -708,26 +709,32 @@ public void handleJsonEvent(final JsonReadEvent jsonReadEvent) {
                     OpenEventApp.postEventOnUIThread(new SessionDownloadEvent(true));
                     break;
                 } case ConstantStrings.SPEAKERS: {
-                    List<Speaker> speakers = objectMapper.readValue(json, new TypeReference<List<Speaker>>() {});
+                    List<Speaker> speakers = objectMapper.readValue(json, objectMapper.getTypeFactory().constructCollectionType(List.class, Speaker.class));
 
                     realmRepo.saveSpeakers(speakers).subscribe();
 
                     OpenEventApp.postEventOnUIThread(new SpeakerDownloadEvent(true));
                     break;
                 } case ConstantStrings.SPONSORS: {
-                    List<Sponsor> sponsors = objectMapper.readValue(json, new TypeReference<List<Sponsor>>() {});
+                    List<Sponsor> sponsors = objectMapper.readValue(json, objectMapper.getTypeFactory().constructCollectionType(List.class, Sponsor.class));
 
                     realmRepo.saveSponsors(sponsors).subscribe();
 
                     OpenEventApp.postEventOnUIThread(new SponsorDownloadEvent(true));
                     break;
                 } case ConstantStrings.MICROLOCATIONS: {
-                    List<Microlocation> microlocations = objectMapper.readValue(json, new TypeReference<List<Microlocation>>() {});
+                    List<Microlocation> microlocations = objectMapper.readValue(json, objectMapper.getTypeFactory().constructCollectionType(List.class, Microlocation.class));
 
                     realmRepo.saveLocations(microlocations).subscribe();
 
                     OpenEventApp.postEventOnUIThread(new MicrolocationDownloadEvent(true));
                     break;
+                }case ConstantStrings.SESSION_TYPES: {
```

Comment:
over here the indentation of generator.py shows some problem however codacy runs successfully.
on my rpivate account link it is also fine: https://github.com/arp95/open-event-android/blob/development/apk-generator/v2/app/generator/generator.py
then what is the problem, is it the editor i am using?
The app seems to run fine now.